touch: refresh (#7966)
tldr-pages/tldr: pages/common/touch.md

# touch

> > Creates files and sets access/modification time.

Reviewer: ```suggestion
> Create files and set access/modification time.
```

> More information: <https://www.gnu.org/software/coreutils/touch>.

- Create specific files:

`touch {{path/to/file1 path/to/file2 ...}}`

- Update file access/modification time to the current one:

`touch --no-create --time={{access|modify}} {{path/to/file1 path/to/file2 ...}}`

> - Update file [t]ime to a specific one:

Reviewer 2: ```suggestion
- Set the file [t]ime to a specific value:
```

> `touch --no-create --time={{mode}} -t {{YYYYMMDDHHMM.SS}} {{path/to/file1 path/to/file2 ...}}`

Reviewer 2: I think `mode` here is pretty confusing.
```suggestion
`touch --no-create --time={{access|modify}} -t {{YYYYMMDDHHMM.SS}} {{path/to/file1 path/to/file2 ...}}`
```

> - Update file time to a specific file time:

Reviewer 2: ```suggestion
- Set file time of a specific file to the time of another file:
```

> `touch --no-create --time={{mode}} --reference={{path/to/file}} {{path/to/file1 path/to/file2 ...}}`

Reviewer 3: This way is more clear
```suggestion
`touch --no-create --time={{access|modify}} --reference={{path/to/file}} {{path/to/file1 path/to/file2 ...}}`
```

Author: @navarroaxel, should I always inline all values for options? What about to do it only when value count is small such as here (just two choices)?

Reviewer 2: This really depends. There are mostly three factors here: The number of options, how intuitive the options are, and how quickly the options can be found in e.g. the man page.

For few options, we can almost always inline them.
If you have many, but intuitive options (like `read`, `write`, `execute`) which everyone knows, we can be generic.
And if the options, can't be easily found somewhere we should also inline them.

But this is highly dependent on the situation. Just think about how it would be the easiest for the user.

- Create multiple files:

> `touch {{path/to/file1 path/to/file2 ...}}`

Reviewer 3: :thinking: This is a simple command, maybe we add a new example with `file{1,2,3}` to explain that bash feature too. wdyt?

Author: It's a good idea to note about shell specific features. 👍🏻 But there is a problem: we have to decide what features are noted about in tldr pages to make pages more standardized. There are too many pages without notion of brace expansion. It is assumed (as I understand) that user adapt suggested syntax to a specific shell. I suggest orient to a POSIX shell and suggest just some of it's features explicitly (as a separate examples or in braces in code descriptions).

[Here][bashisms] you can find bash-specific features. Brace expansion is one of them: and I am not sure we have to stick to Bash shell in some pages. There are other shells too with their specific syntax so it's not clear why not to note about them too. But in this case we are going to make our descriptions very long. 😄 

[bashisms]: https://mywiki.wooledge.org/Bashism

Reviewer: I like our current standard for "multiple files", its clear for users to understand and the people can often almost copy the listed commands.

Author: Well, what about using brace expansion everywhere for arbitrary number of arguments? For package names, user names, file names and etc. 🤔
